app/models/calagator/event/saver.rb

```diff
@@ -34,7 +34,17 @@ def attempt_save?
       end
 
       def spam?
-        evil_robot? || too_many_links?
+        evil_robot? || too_many_links? || ml_flags_as_spam?
+      end
+
+      def get_spam_label
+        GpturkService.get_spam_label("#{params.dig(:event, :title)} #{params.dig(:event, :description)} #{params[:start_date]} at #{params[:start_time]}")
+      end
+
+      def ml_flags_as_spam?
+        if get_spam_label != 1
```

Comment:
I think should get rolled up into the `GpturkService`. It would be better for it to determine if something is spam than for the `Saver`.

Reply:
I could see a `GpturkService.is_this_spam?(text)` method being nice.

Reply:
On it!